iOS Headers not compatible with RN 0.40+

Tried installing in my existing (and also new) project and I'm getting
```
'RCTView.h' not found
'RCTViewManager.h' not found
```

Getting the same here:

```
In file included from /Users/username/reponame/node_modules/lottie-react-native/lib/ios/LottieReactNative/LottieAnimationViewManager.m:9:
/Users/username/reponame/node_modules/lottie-react-native/lib/ios/LottieReactNative/LottieAnimationViewManager.h:9:9: fatal error: 'RCTViewManager.h' file not found
#import "RCTViewManager.h"
        ^
1 error generated.
```

--- a/lib/ios/LottieReactNative.xcodeproj/project.pbxproj
+++ b/lib/ios/LottieReactNative.xcodeproj/project.pbxproj
@@ -269,6 +269,8 @@
 				HEADER_SEARCH_PATHS = (
 					"$(SRCROOT)/../../node_modules/react-native/React/**",
 					"$(SRCROOT)/../../node_modules/react-native/ReactCommon/**",
+					"$(SRCROOT)/../../../react-native/React/**",
+					"$(SRCROOT)/../../../react-native/ReactCommon/**",
 				);
 				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
 				OTHER_LDFLAGS = "-ObjC";
@@ -283,6 +285,8 @@
 				HEADER_SEARCH_PATHS = (
 					"$(SRCROOT)/../../node_modules/react-native/React/**",
 					"$(SRCROOT)/../../node_modules/react-native/ReactCommon/**",
+					"$(SRCROOT)/../../../react-native/React/**",
+					"$(SRCROOT)/../../../react-native/ReactCommon/**",
 				);
 				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
 				OTHER_LDFLAGS = "-ObjC";

